R/get_image.R

```diff
@@ -55,15 +52,15 @@
 #' @export
 get_image <- function(
   file_key,
-  ids,
+  ids = NULL,
```

Comment:
Isn't this a mandatory field?

Reply:
Yep. I think I added the NULL default back for one of the parameters that I removed it from but missed this one.